docs/get-started/upgrading.md

```diff
@@ -18,10 +18,23 @@ If you have your API key stored in the `PL_API_KEY` environment variable you wil
 
 In Version 2, sessions are used to manage all communication with the Planet APIs. This provides for multiple asynchronous connections. For each API, there is a specific client object. This client manages polling and downloading, along with any other capabilities provided by the API.
 
-Each client now requires a `Session` object, which stores connection information and authentication.
+Each client now requires a `Session` object, which stores connection information and authentication and manages an HTTP connection pool.
```

Comment:
Connection pool is a key reason for Session!

Reply:
yep!